Content-type: text/xml

<?xml version="1.0" encoding="utf-8"?>
<testing-report run-id="165" judge-id="95" status="SE" scoring="ACM" run-tests="1" contest-id="3204" real-time-available="yes" max-memory-used-available="yes" correct-available="yes" failed-test="1" time-limit-ms="2000" real-time-limit-ms="5000" marked-flag="no" >
  <uuid>1e7c55d4-f13f-4353-bba6-8ef3a6b7265b</uuid>
  <host>localhost.localdomain</host>
  <cpu-model>Intel(R) Pentium(R) CPU 3550M @ 2.30GHz</cpu-model>
  <cpu-mhz>2294.738</cpu-mhz>
  <compiler_output></compiler_output>
  <tests>
    <test num="1" status="SE" time="283" real-time="304" max-memory-used="18698240" >
  <program-stats-str>{ utime=266, stime=17, ptime=283, rtime=304, maxvsz=18698240, maxrss=19169280, nvcsw=1, nivcsw=44 }</program-stats-str>
      <input size="10">1 2 3 4 5
</input>
      <output size="0"></output>
      <stderr size="293">Can't open '/usr/share/abrt/conf.d/plugins/python3.conf': Operation not permitted
Can't open '/etc/abrt/plugins/python3.conf': Operation not permitted
Traceback (most recent call last):
  File &quot;./000165&quot;, line 8, in &lt;module&gt;
    if a[i-1] &lt; a[i] &gt; a[i+1] :
IndexError: list index out of range
</stderr>
    </test>
  </tests>
</testing-report>
